variables.tf

```diff
@@ -133,4 +133,12 @@ variable "service_endpoints" {
   }
 }
 
+variable "reserved_ips" {
+  description = "Reserved IPs"
```

Comment:
Needs a more descriptive description here. I'm even confused as to what this variable is used for in the root level module. Is it an optional variable to pass existing reserved IPs? The description needs to also explain what happens if nothing is passed.